app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -52,6 +52,10 @@ def perform(_now)
 
   private
 
+  def analytics
+    @analytics ||= Analytics.new(user: AnonymousUser.new, request: nil, session: {}, sp: nil)
```

Comment:
since each proofing enrollment has a user, could we make a new analytics instance per row and plug that in here? Then we can query these events by that user uuid as well

Reply:
added in adf30a49fc80f48a6ee62dcf83da0fac764f2a99